ush/detect_machine.sh

```diff
@@ -76,10 +78,10 @@ elif [[ -d /work ]] ; then
 elif [[ -d /glade ]] ; then
   # We are on NCAR Yellowstone
   MACHINE_ID=cheyenne
-elif [[ -d /lustre && -d /ncrc ]] ; then
-  # We are on GAEA.
+elif [[ -d /gpfs && -d /ncrc ]] ; then
+  # We are on GAEA C5.
   MACHINE_ID=gaea
-elif [[ -d /data/prod ]] ; then
+elif [[ -d /lustre && -d /ncrc ]] ; then
```

Comment:
Please use the same logic as in `ufs-weather-model`.  Please refer to the top of this file if changes are needed in this script.